listenbrainz/webserver/views/playlist_api.py

```diff
@@ -1023,23 +1034,31 @@ def export_playlist_jspf(service):
     """
     user = validate_auth_header()
 
-    if service != "spotify":
+    if service != "spotify" and service != "apple_music":
         raise APIBadRequest(f"Service {service} is not supported. We currently only support 'spotify'.")
```

Comment:
Same as above here, the error text needs to be updated. Please also check other text string in this method for completion.